modules/network-observability-viewing-alerts.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-alert-dashboard_{context}"]
+= Viewing the alert dashboard
+
+You can access alerts for the Network Observability Operator dashboard from the *Alerting* page in the web console.
+
+.Prerequisites
+
+* You have the Network Observability Operator installed.
+* You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects.
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Observe* → *Alerting*.
+. From the *Alerts* list, select the *NetObservLokiError* alert. 
+. Information about the alert is displayed on the page. Alerts are generated in the folling cases:
++
+* If the `flowlogs-pipeline` workload is dropping flows because of Loki errors, such as if the Loki ingestion rate limit has been reached
+* If no flows are ingested for a certain amount of time 
+
+[NOTE]
+====
+The alert also displays a banner in the *Observe*→*Network Traffic* page and the *Home* page.
+====
+
+== Disable health alerts
```

Comment:
@OlivierCazade @jotak I added a section here about disabling health alerts. Can you PTAL? I debated adding this spec to the FlowCollector CRD sample in the Configuring section. I guess it could be added there, but I felt like this parameter is choosing _not to do_ something, and all the parameters we callout in the FlowCollector CRD describe more the functionality, choices _to do_.